code/main/common/int.brs

```diff
@@ -20,3 +20,19 @@ function _adb_isInvalidInt(int as dynamic) as boolean
 
     return false
 end function
+
+function _adb_isInvalidLongInt(number as dynamic) as boolean
+    if number = invalid or (type(number) <> "roLongInteger" and type(number) <> "LongInteger")
+        return true
+    end if
+
+   return false
+end function
+
+function _adb_isPositiveNumber(number as dynamic) as boolean
+    if (_adb_isInvalidInt(number) and _adb_isInvalidLongInt(number)) or number <= 0
```

Comment:
IMO, the functions `_adb_isInvalidInt` and `_adb_isInvalidLongInt` make the code harder to read because they focus on failure cases. It would be more readable if these functions were named `_adb_isValidInt` and `_adb_isValidLongInt`, checking for valid values instead.

This way this function can be changes to 
```
if (_adb_isValidInt(number) or _adb_isValidLongInt(number)) and number > 0 then
    return true
end if

return false
```

Reply:
@praveek updated and cleaned up some unnecessary helpers.